java/serve/src/main/java/io/ray/serve/util/ExampleJaxrsDeployment.java

```diff
@@ -0,0 +1,36 @@
+package io.ray.serve.util;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+
+@Path("example")
+public class ExampleJaxrsDeployment {
```

Comment:
Move to test package, maybe we needle a demo package.

Reply:
Resloved.